src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbBase.java

```diff
@@ -149,15 +149,20 @@ protected MtAmazonDynamoDbContextProvider getMtContext() {
     }
 
     /**
-     * Determines if the table for the given name is a multitenant table associated with this instance.
+     * Determines if the table for the given name is a physical table associated with this instance.
      *
      * @param tableName Name of the table.
-     * @return true if the given table name is a multitenant table associated with this instance, false otherwise.
+     * @return true if the given table name is a physical table associated with this instance, false otherwise.
      */
-    protected boolean isMtTable(String tableName) {
+    protected boolean isPhysicalTable(String tableName) {
```

Comment:
To make this clear that by this we mean a physical table associated with the instance, as opposed to a virtual multitenant table.